frontend/src/components/GameList.vue

```diff
@@ -13,15 +18,22 @@
         title: String,
         author: String,
         description: String,
-        image: String,
+
     },
 
     data: () => ({
       show: false,
+      gameInfo: [1,2,3,4,5,6,7,8]
     }),
 
     components: {
       GameCard,
     },
+
```

Comment:
add another section here called mounted and call getData here.
`mounted() {
  this.getData();
}`